main.css

```diff
@@ -0,0 +1,156 @@
+body {
+	font-family: Georgia, serif;
+	color: white;
+	margin: 0;
+    background-color:  #1B99B9;
+}
+
+header {
+	height: 50px;
+	font-size: 18px;
+	background-color: #708090;
+}
+
+.header-title {
+	float: left;
+}
+
+header div i {
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs